api/app/utils/geospatial.py

```diff
@@ -83,3 +84,21 @@ def raster_mul(tpi_ds: gdal.Dataset, hfi_ds: gdal.Dataset, chunk_size=256) -> gd
             hfi_chunk = None
 
     return out_ds
+
+
+class PointTransformer:
+    """
+    Transforms the coordinates of a point from one spatial reference to another.
+    """
+
+    def __init__(self, source_srs, target_srs):
```

Comment:
nit: some type hints here might be handy

Reply:
Done in: https://github.com/bcgov/wps/pull/3849/commits/af4bd2af7d9680afcf95ab7b6e8537fcb87cdbe9